contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
Wanted to keep it easy to still be compatible if the header order of `project()` arguments gets changed or more fields added to it.  Yes that would be a simpler expression with strictly what's there.